calico-enterprise/getting-started/bare-metal/about.mdx

```diff
@@ -40,94 +40,108 @@ To learn how to restrict traffic to/from hosts and VMs using Calico network poli
+
+    To ensure a secure connection, non-cluster host agents will establish an mTLS connection via the `typhaEndpoint` endpoint. As a result, the external ingress or load balancer must be configured to pass through TCP Layer 4 traffic.
+
+    :::note
+
+    Starting from Calico Enterprise version v3.21.0 Early Preview 2, the `typhaEndpoint` field is required for non-cluster host agents to function correctly. Ensure this filed is properly configured to maintain seamless connectivity.
+
+    :::
+
+    | Field         | Description                                                          | Accepted Values                                                          | Schema |
+    | ------------- | -------------------------------------------------------------------- | ------------------------------------------------------------------------ | ------ |
+    | endpoint      | Location of the log ingestion point for non-cluster hosts. Required. | Any HTTPS URL with a domain name and a port number                       | string |
+    | typhaEndpoint | Location of the Typha endpoint for non-cluster host agent and Typha communication. Required. | Any IP address or domain name with a port number | string |
+
+    ```bash
+    kubectl create -f - <<EOF
+    apiVersion: operator.tigera.io/v1
+    kind: NonClusterHost
+    metadata:
+      name: tigera-secure
+    spec:
+      endpoint: https://ingress-or-load-balancer.example.com:443
+      typhaEndpoint: 5.6.7.8:5473
+    EOF
+    ```
+
+    Wait until the Tigera Manager and non-cluster Typha deployments reach the Available status before proceeding to the next step.
+
+2. Obtain the token for `tigera-noncluster-host` service account.
+
+    ```bash
+    kubectl get secret -n calico-system tigera-noncluster-host -o jsonpath='{.data.token}' | base64 --decode
+    ```
+
+3. Use a text editor to create a kubeconfig file for non-cluster host or VM.
+
+    ```yaml
+    apiVersion: v1
+    kind: Config
+    current-context: noncluster-hosts
+    preferences: {}
+    clusters:
+    - cluster:
+        certificate-authority-data: <your cluster certificate>
+        server: <your cluster server>
+      name: noncluster-hosts
+    contexts:
+    - context:
+        cluster: noncluster-hosts
+        user: tigera-noncluster-host
+      name: noncluster-hosts
+    users:
+    - name: tigera-noncluster-host
+      user:
+        token: <token from previous step>
+    ```
+
+    Take the cluster information from the administrator kubeconfig.
+
+4. Create a [`HostEndpoint` resource](../../reference/host-endpoints/overview.mdx) for each non-cluster host or VM. The `node` and `expectedIPs` fields are required to match the hostname and the expected interface IP addresses.
+
+5. Follow [Configure access to the $[prodname] web console](../../operations/cnx/access-the-manager.mdx) page to configure the $[prodname] web console access.
```

Comment:
This isn't really part of your procedure. If you think it's important to guide users to this information, consider adding it in a 'Next steps' section.

Reply:
Step 5 is removed.